javascript/features/communication_commands/direct_communication_commands.js

```diff
@@ -20,6 +22,8 @@ export class DirectCommunicationCommands {
     communication_ = null;
     nuwani_ = null;
     playground_ = null;
+    
+    muteManager_ = new MuteManager();
```

Comment:
nit: delete, also enables you to delete the `import { MuteManager }` line

Reply:
Fixed as of b10fd3ed14f79bc2e46618c53df2363d2a7ba280